python/ml4ir/applications/ranking/features/feature_fns/string.py

```diff
@@ -158,3 +170,172 @@ def call(self, inputs, training=None):
         query_type_vector = self.categorical_vector_op(query_type, training=training)
 
         return query_type_vector
+
+
+class QueryEmbeddingVector(BaseFeatureLayerOp):
```

Comment:
@mohazahran As this is only supporting Glove embeddings at the moment, should we just name it to GloveQueryEmbedding?